handbook/ce/ce.md

```diff
@@ -0,0 +1,41 @@
+# Customer Engineers
+
+## Our ethos
+
+Customer Engineers at Sourcegraph are a combination of pre-sales sales engineering and post-sales customer success. We own the relationship with our developer users and champions at our customers and prospects. We serve as the technical experts during the sales cycle and help enable our customers throughout our journey together.
+
+We show up for our customers, open source users, and teammates by living up to our Sourcegraph values and handbook, as well as in the following ways:
+
+- We deeply know and understand our customers and their needs; we enable them and guide them
+- We teach our customers new and better ways to get value from Sourcegraph as thought leaders and evangelists
+- We advocate on behalf of our customers internally to product and engineering
```

Comment:
Given our conversations in recent days, it feels like this maybe could be reworded, too ... to reflect about doing what is in the best interest of our customers. I think Quinn's thoughts in the Uber thread make it clearer what this means, but maybe also not clear enough. I could just see this creating friction if folks ask CE's questions to understand, maybe even challenge and that feeling like they are alone in advocating ... if that makes sense?

Reply:
@virginiaulrich could it be "We, in collaboration with AEs and CSEs advocate on behalf of our customers internally to product and engineering" something like that?

Reply:
I was more so thinking here of advocation in terms of voice of the customer, feedback, feature requests, etc.

How about from: We advocate on behalf of our customers internally to product and engineering
To: We serve as the voice of the customer to internal teams, providing key insights and feedback, and raising feature requests.

Also open to other suggestions!

Reply:
I like your v2, Aimee!